src/pages/Apply.jsx

```diff
@@ -0,0 +1,144 @@
+import { useParams } from "react-router-dom";
+import { useState } from "react";
+import PhoneInput from "react-phone-input-2";
+import "react-phone-input-2/lib/style.css";
+
+const Apply = () => {
+  const { slug } = useParams();
+  const [formData, setFormData] = useState({
+    name: "",
+    email: "",
+    phone: "",
+    address: "",
+    relocation: "",
+    startDate: "",
+    immigrationStatus: "",
+    workPermitExpiry: "",
+    resume: null,
+  });
+
+  const handleChange = (e) => {
+    const { name, value } = e.target;
+    setFormData({ ...formData, [name]: value });
+  };
+
+  const handlePhoneChange = (phone) => {
+    setFormData({ ...formData, phone });
+  };
+
+  const handleFileChange = (e) => {
+    setFormData({ ...formData, resume: e.target.files[0] });
+  };
```

Comment:
We'll need to upload this file to an S3 bucket. I'll share the stuff for that tomorrow.